src/rtlib/unix/sys_dylib.c

```diff
@@ -17,11 +17,11 @@ FBCALL void *fb_DylibLoad( FBSTRING *library )
 							  "./lib%s.so",
 							  NULL };
 
-	// Just in case the shared lib is an FB one, temporarily reset the
```

Comment:
Newer C compilers should accept `// comments' this.  For portability, all C compilers will accept `/* comments */'.  Regardless, the comments should not be removed.